src/rng.jl

```diff
@@ -0,0 +1,104 @@
+using Random123
+using Random
+using Distributions
```

Comment:
Could these be moved to src/AdvancedPS.jl?

Reply:
Yeah, I'd prefer to see all the imports in `src/AdvancedPS.jl`.